CONTRIBUTING.md

```diff
@@ -253,6 +253,13 @@ out of convenience.
     fixing a typo in an ID still requires some description on how you checked
     that the new ID takes effect.
 
+> [!TIP] When working on a pull request for Matter SDK refrain from using the
```

Comment:
```suggestion
> [!TIP]
> When working on a pull request for Matter SDK refrain from using the
```

Reply:
https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts

Reply:
This was reformatted by restyled. We will see if it will pass when we restore the old syntax.